index.js

```diff
@@ -19,12 +19,22 @@ const linkify = (href, options) => createHtmlElement({
 // Get DOM node from HTML
 const domify = html => document.createRange().createContextualFragment(html);
 
-const getAsString = (string, options) => string.replace(urlRegex(), match => linkify(match, options));
+const isTruncated = (url, peek) =>
+	url.endsWith('...') // `...` is a matched by the URL regex
+	|| peek.startsWith('…'); // `…` can follow the match
```

Comment:
This change slightly extracted the logic into the code itself (the `peek` extraction) but this actually allowed simplifying this feature substantially as well as logic reuse.